PopupBridge/src/main/java/com/braintreepayments/api/internal/AppHelper.kt

```diff
@@ -0,0 +1,23 @@
+package com.braintreepayments.api.internal
+
+import android.content.Context
+import android.content.pm.PackageManager
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+internal class AppHelper {
+
+    companion object {
+        private const val NO_FLAGS = 0
+    }
+
+    fun isAppInstalled(context: Context, packageName: String): Boolean {
+        val packageManager: PackageManager = context.packageManager
+        try {
+            packageManager.getApplicationInfo(packageName, NO_FLAGS)
+            return true;
+        } catch (_: PackageManager.NameNotFoundException) {
+            return false;
+        }
```

Comment:
```suggestion
        return try {
            packageManager.getApplicationInfo(packageName, NO_FLAGS)
            true
        } catch (_: PackageManager.NameNotFoundException) {
            false
        }
```

Nits - we can lift the return out of the try catch and remove the semicolons.